server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -264,89 +260,163 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
+		);
+		const socketSubmitOpThrottler = configureThrottler(submitOpThrottleConfig);
+		const submitSignalThrottleConfig = utils.getThrottleConfig(
+			config.get("alfred:throttling:generalCluster:submitSignals"),
+		);
+		const socketSubmitSignalThrottler = configureThrottler(submitSignalThrottleConfig);
 
 		const restClusterThrottlers = new Map<string, core.IThrottler>();
 		restClusterThrottlers.set(Constants.createDocThrottleIdPrefix, restCreateDocThrottler);
 		restClusterThrottlers.set(Constants.getDeltasThrottleIdPrefix, restGetDeltasThrottler);
 		restClusterThrottlers.set(Constants.getSessionThrottleIdPrefix, restGetSessionThrottler);
-
-		// Socket Connection Throttler
-		const socketConnectionThrottleConfigPerTenant = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerTenant"),
+		restClusterThrottlers.set(
+			Constants.socketConnectionsThrottleIdPrefix,
+			socketConnectClusterThrottler,
 		);
-		const socketConnectTenantThrottler = configureThrottler(
-			socketConnectionThrottleConfigPerTenant,
+		restClusterThrottlers.set(Constants.submitOpsThrottleIdPrefix, socketSubmitOpThrottler);
+		restClusterThrottlers.set(
+			Constants.submitSignalThrottleIdPrefix,
+			socketSubmitSignalThrottler,
 		);
 
-		const socketConnectionThrottleConfigPerCluster = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerCluster"),
+		// Tenant Group 1 Rest API Throttlers
+		const restApiCreateDocThrottleConfigGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:tenantGroup1:createDoc"),
 		);
-		const socketConnectClusterThrottler = configureThrottler(
-			socketConnectionThrottleConfigPerCluster,
+		const restCreateDocThrottlerGroup1 = configureThrottler(
+			restApiCreateDocThrottleConfigGroup1,
 		);
-
-		// Socket SubmitOp Throttler
-		const submitOpThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:submitOps"),
+		const restApiGetDeltasThrottleConfigGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:tenantGroup1:getDeltas"),
+		);
+		const restGetDeltasThrottlerGroup1 = configureThrottler(
+			restApiGetDeltasThrottleConfigGroup1,
+		);
+		const restApiGetSessionThrottleConfigGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:tenantGroup1:getSession"),
+		);
+		const restGetSessionThrottlerGroup1 = configureThrottler(
+			restApiGetSessionThrottleConfigGroup1,
+		);
+		const socketConnectionThrottleConfigPerClusterGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:tenantGroup1:socketConnections"),
+		);
+		const socketConnectClusterThrottlerGroup1 = configureThrottler(
+			socketConnectionThrottleConfigPerClusterGroup1,
+		);
+		const submitOpThrottleConfigGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:tenantGroup1:submitOps"),
+		);
+		const socketSubmitOpThrottlerGroup1 = configureThrottler(submitOpThrottleConfigGroup1);
```

Comment:
we can just call this submitOpThrottlerGroup1. same for signals

Reply:
Sounds good! Done. :)